package.json

```diff
@@ -90,7 +90,7 @@
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
-    "lzma-native": "^1.5.2",
+    "lzma-native": "^2.0.2",
```

Comment:
We should stop explicitly depending on it altogether, in my opinion. If there's a node version with a buggy node-gyp, we should either move to a different version, or install a non-buggy one globally, I think. But both cases shouldn't be necessary anymore when we're on node 8.